src/user_interface.py

```diff
@@ -610,19 +618,28 @@ def __confirm(self):
 
     def __show_help(self):
         helpfile = str(Path(__file__).parent / "assets" / "manual.pdf")
-        subprocess.Popen([helpfile], shell=True)
+        if platform.system() in ('Darwin','Linux'):
+            subprocess.Popen(['open',helpfile])
+        else:
+            subprocess.Popen([helpfile], shell=True)
 
     def __show_sheet(self):
         if (self.form_variant == FormVariantSelection.VARIANT_75_Q):
             helpfile = str(
                 Path(__file__).parent / "assets" /
                 "multiple_choice_sheet_75q.pdf")
-            subprocess.Popen([helpfile], shell=True)
+            if platform.system() in ('Darwin','Linux'):
+                subprocess.Popen(['open', helpfile])
+            else:
+                subprocess.Popen([helpfile], shell=True)
         elif (self.form_variant == FormVariantSelection.VARIANT_150_Q):
             helpfile = str(
                 Path(__file__).parent / "assets" /
                 "multiple_choice_sheet_150q.pdf")
-            subprocess.Popen([helpfile], shell=True)
+            if platform.system() in ('Darwin','Linux'):
+                subprocess.Popen(['open', helpfile])
+            else:
+                subprocess.Popen([helpfile], shell=True)
```

Comment:
Could you please remove all the other changes from this pull request and only include these three changes? That way I can just merge this fix and we can address the scrolling changes in https://github.com/iansan5653/open-mcr/pull/74.

Reply:
I will cherry pick as soon as I get a chance. We are desperate here. So, I am focused on getting something we can use first.

Reply:
Unless I missed something the commits below resolve this by removing  the scrolling commits.